package.json

```diff
@@ -17,8 +17,11 @@
         "t": ""
     },
     "devDependencies": {
+        "@codemirror/state": "^6.2.0",
+        "@codemirror/view": "^6.9.1",
         "@types/debug": "^4.1.7",
         "@types/diff": "^5.0.2",
+        "@types/hogan.js": "^3.0.1",
```

Comment:
What's the reason for this?

Reply:
@Vinzent03 
Because without this, we will get a compile error `error TS7016: Could not find a declaration file for module 'hogan.js'.`. hogan.js is used in diff2html.

Reply:
Ah okay, thanks for explanation